bench/hono.bench.mts

```
@@ -0,0 +1,88 @@
+        route: "/static/index.html",
+    },
+]
+const trieRouter = new TrieRouter()
+
+summary(() => {
+    bench("createRouter insert", () => {
+        createRouter(routes as RouteEndpoint[])
+    })
+
+    bench("TrieRouter insert", () => {
+        routes.forEach((route) => trieRouter.add({ method: route.method, route: route.route } as RouteEndpoint))
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)